### ISSUE TITLE
ADManagedServiceAccount: The term 'Get-LocalizedData' is not recognized

### ISSUE THREAD

**Issue by alexmartinio** [dsccommunity/ActiveDirectoryDsc#612]
#### Details of the scenario you tried and the problem that is occurring
Trying to use the ADManagedServiceAccount resource to create a service account with the current pre-release of the DSC module fails with an `ObjectNotFound` exception. 

Importing the module manually also fails. 

#### Verbose logs showing the problem
```
Importing module MSFT_ADManagedServiceAccount failed with error - The term 'Get-LocalizedData' is not recognized as the name of a cmdlet, function, script file, or operable program. Check the spelling of the name, or if a path was included, verify that the path is correct and try again.
```

#### Suggested solution to the issue
Unknown

#### The DSC configuration that is used to reproduce the issue (as detailed as possible)
```powershell
PS C:\Users\Administrator> Import-Module ActiveDirectoryDsc
Get-LocalizedData : The term 'Get-LocalizedData' is not recognized as the name of a cmdlet, function, script file, or
operable program. Check the spelling of the name, or if a path was included, verify that the path is correct and try
again.
At C:\Program Files\WindowsPowerShell\Modules\ActiveDirectoryDsc\6.1.0\Modules\ActiveDirectoryDsc.Common\ActiveDirector
yDsc.Common.psm1:7 char:25
+ $script:localizedData = Get-LocalizedData -DefaultUICulture 'en-US'
+                         ~~~~~~~~~~~~~~~~~
    + CategoryInfo          : ObjectNotFound: (Get-LocalizedData:String) [], CommandNotFoundException
    + FullyQualifiedErrorId : CommandNotFoundException
```

#### The operating system the target node is running
```
OsName               : Microsoft Windows Server 2019 Datacenter
OsOperatingSystemSKU : DatacenterServerEdition
OsArchitecture       : 64-bit
WindowsVersion       : 1809
WindowsBuildLabEx    : 17763.1.amd64fre.rs5_release.180914-1434
OsLanguage           : en-US
OsMuiLanguages       : {en-US}
```

#### Version and build of PowerShell the target node is running
```
Name                           Value
----                           -----
PSVersion                      5.1.17763.1007
PSEdition                      Desktop
PSCompatibleVersions           {1.0, 2.0, 3.0, 4.0...}
BuildVersion                   10.0.17763.1007
CLRVersion                     4.0.30319.42000
WSManStackVersion              3.0
PSRemotingProtocolVersion      2.3
SerializationVersion           1.1.0.1
```

#### Version of the DSC module that was used
`6.1.0-preview0002`

**Comment by alexmartinio** [comment]
It seems `6.1.0-preview0001` also had this problem. Will revert back to `6.0.1`.

**Comment by X-Guardian** [comment]
Hi @alexmartinio. Thanks for reporting this, I'll take a look.

The last big refactor of the `ADManagedServiceAccount` resource was released in version 5, so there is nothing to be released in 6.1.0 that changes this resource, so no need to use the preview versions.
